src/core/server/saved_objects/mappings/types.ts

```diff
@@ -147,13 +146,15 @@ export interface SavedObjectsCoreFieldMapping {
  * See {@link SavedObjectsFieldMapping} for documentation.
  *
  * Note: this type intentially doesn't include a type definition for defining
- * the `dynamic` mapping parameter. Saved Object fields should always inherit
- * the `dynamic: 'strict'` paramater. If you are unsure of the shape of your
- * data use `type: 'object', enabled: false` instead.
+ * `dynamic: true`. To limit the number of mapping fields, Saved Object types
+ * should only ever be set to `dynamic: false` or inherit the default
+ * `dynamic: 'strict'` from the top-level mapping.
  *
  * @public
  */
 export interface SavedObjectsComplexFieldMapping {
+  dynamic?: false;
```

Comment:
`false | 'strict'` for consistency with `SavedObjectsTypeMappingDefinition.dynamic` ?

Reply:
I thought it doesn't make sense to ever specify `'strict'` but in theory you could have a `dynamic: false` on one level but want a `dynamic: 'strict'` on one of the child levels, so it makes sense to add this.